src/compiler/program.ts

```diff
@@ -1589,6 +1589,7 @@ namespace ts {
                 // synthesize 'import "tslib"' declaration
                 const externalHelpersModuleReference = createLiteral(externalHelpersModuleNameText);
                 const importDecl = createImportDeclaration(/*decorators*/ undefined, /*modifiers*/ undefined, /*importClause*/ undefined);
+                importDecl.transformFlags |= TransformFlags.NeverApplyImportHelper;

```

Comment:
The only place that should be setting `transformFlags` is `computeTransformFlagsForNode` in _binder.ts_ (the only notable exception is `convertIterationStatementBodyIfNecessary` in _es2015.ts_, which I'd eventually like to fix as well). This seems like it should be on `EmitFlags` instead.